.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+
+💐
+
+/cc @zendesk/dingo
+
+### Description
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
+
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
+
+### Risks
+* [RUNTIME] Does this change touch the apps runtime?
+* [HIGH | medium | low] Does it work across browsers?
+* [HIGH | medium | low] Are there any performance implications?
+* [HIGH | medium | low] Any security risks?
+* [HIGH | medium | low] What features does this touch?
```

Comment:
I wonder if these would work better as part of the checklist rather than example risks? It's pretty easy to get in the habit of just automatically deleting the bottom part of the template.

Reply:
Yeah, good idea.